src/app/components/navbar/loginSignUpDialog.html

```diff
@@ -0,0 +1,72 @@
+<button mat-icon-button style="margin-left: 660px; margin-top: -50px; margin-bottom: -100px;" aria-label="Close"
```

Comment:
Wrong place for file.
Please create separate component for it. Under routes directory, since signup/login can also be different route rather then only dialog box